src/i18n/index.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { createIntl, createIntlCache, RawIntlProvider } from 'react-intl';
+
+import messages from './translations';
+
+function normalizedLanguage(language?: string) {
+  switch (language) {
+    case 'en':
+      return 'en';
+
+    case 'zh':
+    default:
+      return 'zh';
```

Comment:
In this library, default should be `en`
We can set another default in caller application